about.html

```diff
@@ -4,9 +4,110 @@
+  </script>
+  <link rel="icon" type="image/png" sizes="32x32" href="assets/icon/valorant-logo.svg" />
+  <link rel="stylesheet" href="./css/about/about.css">
+  <title>Valorant Master</title>
 </head>
 <body>
-  
+  <div class="top-bar">
+    <i class="fab fa-facebook-f"></i>
+    <i class="fab fa-twitter"></i>
+    <p>Logout</p>
+    <p>Mypage</p>
+    <p>Spanish</p>
+  </div>  
+  <header>
+    <nav>
+      <a href="index.html" class="link">
+        <img src="assets/icon/valorant-menu-icon.svg" alt="valorant logo">
+      </a>
+      <ul class="menu ">
+          <li>
+              <a href="index.html" class="link">Home</a>
+          </li>
+          <li>
+              <a href="#description" class="link">Description</a>
+          </li>
+          <li>
+              <a href="#location" class="link">This Year Location</a>
+          </li>
+          <li>
+              <a href="#lastTournaments" class="link">Past Tournaments</a>
+          </li>
+      </ul>
+      <i class="fas fa-bars menu-icon" id="hamburguerIcon"></i>
+    </nav>
+
+    <div id="sideBar" class="sidebar">
+      <!-- Modal content -->
+        <div class="sidebar-content">
+            <span class="close-sidebar">&times;</span>
+            <ul>
+                <li><a href="index.html" class="sidebarNavigation" >Home</a></li>
+                <li><a href="#description"   class="sidebarNavigation">Description</a></li>
+                <li><a href="#location" class="sidebarNavigation">This Year Location</a></li>
+                <li><a href="#lastTournaments" class="sidebarNavigation">Past Tournaments</a></li>
+            </ul>
+        </div>
+    </div>
+  </header>
+  <main>
+
+    <section id="description" class="description">
+      <div class="description-title">
+        <p>"Hello! Valorant Players"</p>
+        <h1>VCT VALORANT MASTERS TOURNAMENT 2021</h1>
+      </div>
+      <div class="description-text">
+        <p>Let's start with Challengers. Regions around the world will compete in Challengers events in three stages throughout the year. Each stage features two Challengers main events, which any team from the region can qualify for through open qualifiers. At the main events, teams compete for a place in the Challenger qualifiers. Teams that make it to and win the Challengers qualifiers will qualify for a global competition at the Masters events.</p>
+        <h2>Please Contact us to register your team</h2>
+        <p class="email">valorantplays@riotgames.com</p>
+      </div>
+    </section>
+
+    <section id="location" class="location">
+      <h2>Valorant Master Location 2021</h2>
+      <p>This location in Island was choose to fullfil all the requirements of the players and to fin a middle point between all teams.</p>
+      <div class="img-container">
+        <img src="./assets/images/stadium.jpeg" alt="stadium">
+      </div>
+    </section>
+
+    <section id="lastTournaments" class="lastTournaments">
+      <h1>See the past Tournaments</h1>
```

Comment:
- [ ] Please refactor the positioning of this `<h1>` tag. It should be centered. This is how it looks like right now in the demo 👇🏼 

![center_this](https://user-images.githubusercontent.com/45002654/125000019-15a7e280-e058-11eb-8bc7-c1fa1ea5ce14.PNG)